docs/technical-overview/architecture.md

```diff
@@ -112,6 +112,13 @@ Article has many comments and taggings through the acts-as-taggable gem, belongs
 to a single user (and possibly an organization), and is the core unit of
 content.
 
+## Collections (or series)
+
+Collections are called Series on the frontend of the website. 
+Collections group articles so that groups of articles can be read as a series of posts.
+Collections have not been fully developed as there is not the option to edit or 
+delete series/collections yet.
```

Comment:
I don't think we should list missing features in the documentation though.

Maybe we can be a bit more colloquial as well:

`Although the source code refers to them as "collections", groups of articles are referred throughout the user interface as "series". They represent a collection of articles related to the same topic, indeed, a series`

or something like that. Deferring to @jacobherrington for the grammar check on my suggestion :P

Reply:
Close 😉

`Although the source code refers to them as "collections" groups of articles are referred to, throughout the user interface, as "series". They represent a collection of articles related to the same topic, indeed, a series`